cookbooks/cdo-ruby/recipes/2.0-remove.rb

```diff
@@ -0,0 +1,25 @@
+# Uninstall Ruby 2.0.
```

Comment:
I'm not sure if this `-remove` recipe pattern is a best-practice for Chef, but I found it helpful in this case because we're moving to a different Chef configuration for the > 2.0 versions and it was useful to do some extra cleanup on our manually-configured 2.0 installation for a clean transition. In the future, all we should need to do to upgrade to future versions published to the Brightbox repo is to change the version attribute and Apt should upgrade to the new package version properly.

If for example Brightbox stops maintaining/publishing new Ruby versions and we need to switch to a different PPA for future upgrades, we might want to add a similar `remove` recipe to remove the old PPA before we add the new one (though the cleanup might not be strictly necessary, we could always run a test on the transition first and see if it works without any extra changes)